### ISSUE TITLE
Multi-line tab strings

### ISSUE THREAD

**Issue by GLodi** [hackware1993/MagicIndicator#12]
Is there any way to set the titles with multi-line strings? Something like this:
<img width="459" alt="ss" src="https://cloud.githubusercontent.com/assets/4631114/19445541/f6d7ff34-9494-11e6-9ccb-3dba3ddd27b0.png">

**Comment by hackware1993** [comment]
Sure, you can use CommonPagerTitleView to load a custom layout xml which contains multi-line strings，just reference to LoadCustomLayoutExampleActivity.java in demo.

**Comment by GLodi** [comment]
It worked, thanks :D
I've immediately found a problem though. I've built something very similar to what you see in the picture using a CommonPagerTitleView. My issue is: if I have set a view in the ViewPager that corresponds to "Stories", and swipe twice to the right (showing the "Following" view), than the view under "Stories" disappears. How can I prevent the views set in the ViewPager from disappearing?
